src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,57 @@
+package duke;
+
+import duke.command.Command;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.tasks.Task;
+import duke.ui.Ui;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+
+/**
+ * This program is a chatbot that helps keep track of various tasks.
+ *
+ * @author Lethicia Renissa Santoso (G12)
+ */
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+
+    public Duke(String filePath) {
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (DukeException e) {
+            Ui.showErrorMessage(e);
+            tasks = new TaskList(new ArrayList<Task>());
+        }
+    }
+
+    /**
+     * runs Duke.
+     */
+    public void run() {
+        Ui.showWelcomeMessage();
+        Scanner scan = new Scanner(System.in);
```

Comment:
This is very good coding standard adherance. The spaces between the equals sign is very nice.